source/reference/api/start.txt

```diff
@@ -112,6 +112,15 @@ Request Body Parameters
 
        .. versionadded:: 1.3.0
 
+   * - ``documentFilter``
```

Comment:
How will we only show this in the beta docs?

Reply:
felt it'd be helpful to mention this here, but adding a note that it's only available on beta.